l10n_ch_base_bank/views/invoice.xml

```diff
@@ -9,11 +9,24 @@
       <field name="inherit_id" ref="account.invoice_form"/>
       <field name="arch" type="xml">
         <field name="date_due" position="after">
-          <field name="reference_type" nolabel="1"  attrs="{'readonly':[('state','!=','draft')]}"/>
-          <field name="reference" nolabel="1"  attrs="{'readonly':[('state','!=','draft')]}"/>
+          <field name="reference_type" nolabel="1"/>
```

Comment:
Why was it removed ? Was it for test purpose ? I think we want to keep the readonly state once the invoice has been validated.